spec/controllers/tasks_controller_spec.rb

```diff
@@ -4,14 +4,31 @@
 RSpec.describe TasksController, type: :controller do
 
     describe "POST#create" do
-        it" returns http success " do
+        it "returns http success " do
             post :create, params: { task: {:description => "test1"}}
             expect(response).to have_http_status(:success)
         end
 
-        it" returns an error if the task is not valid " do
+        it "returns an error if the task is not valid " do
             post :create, params: { task: {:description => nil}}
             expect(JSON.parse(response.body)).to eq("error" => "The task couldn't be added. Try again!")
         end
     end 
+
+    describe "GET#index" do
```

Comment:
I think a valuable test here is to create `x` tasks and assert the response contains `x` tasks.

Reply:
Thanks for suggesting this.